styles/asciidoc.atom-text-editor.less

```diff
@@ -59,7 +59,12 @@ atom-text-editor, :host {
       }
 
       &.heading.blocktitle,
-      &.substitution {
+      &.substitution{
+        color: @syntax-text-color-unobtrusive;
+      }
+      &.meta.attrlist {
+        font-weight: normal;
```

Comment:
Styling bad practice.
I see why you do that but you must change the captures group instead of doing that.

ex:

```coffee
  name: 'markup.emphasis.unconstrained.asciidoc'
  # now including the underscores in the match
  match: '(?<!\\\\\\\\)(\\[(?:[^\\]]+?)\\])?((__)((?!_).+?)(__))'
  captures:
    1: name: 'markup.meta.attrlist.asciidoc'
    2: name: 'markup.italic.emphasis.asciidoc'
    3: name: 'punctuation.definition.entity.asciidoc'
    4: name: 'punctuation.definition.entity.asciidoc'
```